File: myblog-server/src/main/java/com/site/blog/controller/PageController.java
package com.site.blog.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

/**
 * @author Yangzhengqian
 * @description
 * @date:Created time 2021/8/12 17:30
 * @modified By:
 */
@Controller
public class PageController {

    @GetMapping("/index")
    public String index (){
        return  "index";
    }
}

File: myblog-kt/src/main/kotlin/com/site/blog/controller/PageController.kt
package com.site.blog.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.ResponseBody

/**
 * @author Yangzhengqian
 * @description
 * @date:Created time 2021/8/12 17:30
 * @modified By:
 */
@Controller
class PageController {
    @GetMapping("/index")
    fun index(): String {
        return "index"
    }
    @GetMapping("")

    fun home() :String{
        return "index"
    }
}